florestad/src/main.rs

```diff
@@ -271,6 +277,10 @@ fn run_with_ctx(ctx: Ctx) {
     #[cfg(not(feature = "compact-filters"))]
     let cfilters = None;
 
+    let connect = ctx
+        .connect
+        .map(|host| LocalAddress::from_str(&host).unwrap());
```

Comment:
maybe treat this error with `.expect` instead of just unwrapping

Reply:
https://github.com/Davidson-Souza/Floresta/pull/119/commits/f360c799ae000bc77c56a937c55d54689f78a188 adds better error handling. I don't think we should panic if the address is invalid, just ignore it. Thoughts?

Reply:
hmm, that's tricky but I think that if this flag connects to this guy exclusively, then it should panic if the address is invalid. If it was inclusive, then it should just ignore and connect to anyone as usual.

Reply:
Yeah, I guess you're right. Changed it to die if a bad address is given.